operator/src/main/resources/application-rhbk.properties

```diff
@@ -0,0 +1,7 @@
+operator.keycloak.pod-labels."com.company"=Red_Hat
+operator.keycloak.pod-labels."rht.prod_name"=Red_Hat_Runtimes
+operator.keycloak.pod-labels."rht.prod_ver"=${SSO_QUARTER:sso_quarter_placeholder}
```

Comment:
@vmuzikar Should also be updated, right?

Reply:
Explained in the private conversation.

Reply:
Discussed offline, thanks